tests/test_equivalence.py

```diff
@@ -20,13 +21,43 @@
 from fasm2bels.fasm2bels import main
 
 
-test_names = ["add32"]
+test_names = ["add32","alu"]
 
 
 def unpack_tar(tar_file):
     tar = tarfile.open(name=tar_file, mode="r:gz")
     tar.extractall(path=os.path.dirname(tar_file))
 
+def createGoldenfile(fileName):
```

Comment:
Would be good to stick with snake_case instead of camelCase to name functions, similarly to the other ones, and this applies also to the variable names